nextflow.config

```diff
@@ -195,43 +194,10 @@ process {
 
 
 profiles {
-  //
-  // The standard profile uses the local executor, in which processes are simply
-  // launched as normal processes on the local machine. By default the local
-  // executor uses the number of CPU cores to limit how many processes are run
-  // in parallel.
-  //
-  standard {
-    process.executor = "local"
-  }
-
-  // A profile for testing on the Travis.CI testing platform.
-  travis {
+  docker {
```

Comment:
I thought this section was here for Travis CI integration.  Why is this changed?

Reply:
Nevermind I see this was just a rearrangement.